x-pack/plugins/alerting/common/routes/r_rule/response/schemas/v1.ts

```diff
@@ -8,40 +8,157 @@
   freq: schema.maybe(
-    schema.oneOf([
-      schema.literal(0),
-      schema.literal(1),
-      schema.literal(2),
-      schema.literal(3),
-      schema.literal(4),
-      schema.literal(5),
-      schema.literal(6),
-    ])
-  ),
-  until: schema.maybe(schema.string()),
-  count: schema.maybe(schema.number()),
-  interval: schema.maybe(schema.number()),
+    schema.oneOf(
+      [
+        schema.literal(0),
+        schema.literal(1),
+        schema.literal(2),
+        schema.literal(3),
+        schema.literal(4),
+        schema.literal(5),
+        schema.literal(6),
+      ],
+      {
+        meta: {
+          description:
+            'Indicates frequency of the rule. Options are YEARLY, MONTHLY, WEEKLY, DAILY.',
+        },
+      }
+    )
+  ),
+  until: schema.maybe(
+    schema.string({
+      meta: {
+        description: 'Recur the rule until this date.',
+      },
+    })
+  ),
+  count: schema.maybe(
+    schema.number({
+      meta: {
+        description: 'Number of times the rule should recur until it stops.',
+      },
+    })
+  ),
+  interval: schema.maybe(
+    schema.number({
+      meta: {
+        description:
+          'Indicates the interval of frequency, e.g. 1 and YEARLY is every 1 year, 2 and WEEKLY is every 2 weeks.',
+      },
+    })
+  ),
   wkst: schema.maybe(
-    schema.oneOf([
-      schema.literal('MO'),
-      schema.literal('TU'),
-      schema.literal('WE'),
-      schema.literal('TH'),
-      schema.literal('FR'),
-      schema.literal('SA'),
-      schema.literal('SU'),
-    ])
-  ),
-  byweekday: schema.maybe(schema.arrayOf(schema.oneOf([schema.string(), schema.number()]))),
-  bymonth: schema.maybe(schema.arrayOf(schema.number())),
-  bysetpos: schema.maybe(schema.arrayOf(schema.number())),
-  bymonthday: schema.maybe(schema.arrayOf(schema.number())),
-  byyearday: schema.maybe(schema.arrayOf(schema.number())),
-  byweekno: schema.maybe(schema.arrayOf(schema.number())),
-  byhour: schema.maybe(schema.arrayOf(schema.number())),
-  byminute: schema.maybe(schema.arrayOf(schema.number())),
-  bysecond: schema.maybe(schema.arrayOf(schema.number())),
+    schema.oneOf(
+      [
+        schema.literal('MO'),
+        schema.literal('TU'),
+        schema.literal('WE'),
+        schema.literal('TH'),
+        schema.literal('FR'),
+        schema.literal('SA'),
+        schema.literal('SU'),
+      ],
+      {
+        meta: {
+          description:
+            'Indicates the start of week, defaults to Monday. The following, if not provided, will be automatically derived from the dtstart.',
+        },
+      }
+    )
+  ),
+  byweekday: schema.maybe(
+    schema.arrayOf(
+      schema.oneOf([schema.string(), schema.number()], {
+        meta: {
+          description:
+            'Indicates day(s) of the week to recur, OR nth-day-of-month strings, e.g. "+2TU" second Tuesday of month, "-1FR" last Friday of the month, which will get internally converted to a byweekday/bysetpos combination.',
```

Comment:
We generally don't use "(s)" since I think it doesn't translate well. 
```suggestion
            'Indicates the days of the week to recur or else nth-day-of-month strings. For example, "+2TU" second Tuesday of month, "-1FR" last Friday of the month, which are internally converted to a `byweekday/bysetpos` combination.',
```